sdk/eventhub/azure-eventhub/stress/Dockerfile

```diff
@@ -1,6 +1,7 @@
-# syntax=docker/dockerfile:1
-
-FROM python:3.8-slim-buster
+# internal users should provide MCR registry to build via 'docker build . --build-arg REGISTRY="mcr.microsoft.com/mirror/docker/library/"'
+# public OSS users should simply leave this argument blank or ignore its presence entirely
+ARG REGISTRY=""
```

Comment:
It is so awesome. Do we publish to mcr somewhere when we pass the mcr register parameter or it is just using the mcr proxy?